rivers/yahoo-finance-stats/parser.js

```diff
@@ -0,0 +1,41 @@
+var _ = require('lodash'),
+    moment = require('moment-timezone');
+
+module.exports = function(body, options, temporalDataCallback, metaDataCallback) {
+    var config = options.config,
+        data = JSON.parse(body);
+
+    moment.tz.setDefault(config.timezone);
+
+    var dtarray = data.query.created.split('T'),
+        dateString = dtarray.shift(),
+        timeString = dtarray.shift().split('Z').shift(),
+        date = moment(dateString + ' ' + timeString, 'YYYY-MM-DD HH:mm:ss'),
+        timestamp = date.unix(),
+        res = data.query.results.quote,
+        symbol = res.symbol,
+        EBITDAStr = res.EBITDA,
+        fieldValues;
+
+    var EBDITA = undefined;
```

Comment:
Capital `EBDITA` implies a constant, but this is not a constant. I suggest `ebdita`.